lua/scratch/config.lua

```diff
@@ -148,24 +169,20 @@ local function initProcess(force)
     return
   end
 
-  -- write the scratch_file_dir into CONFIG_FILE_PATH file
-  local dir_path = vim.fn.fnamemodify(CONFIG_FILE_FLAG_PATH, ":h")
-  if vim.fn.isdirectory(dir_path) == 0 then
-    vim.fn.mkdir(dir_path, "p")
+  -- check if config already exists
+  if vim.fn.filereadable(configFilePath) == 1 then
+    vim.ui.input({
+      prompt = "Found existing config at " .. configFilePath .. ".\n Do you want overwrite it?(Yes/No)",
+    }, function(input)
+      if input == "Yes" then
```

Comment:
I think this is more standard:
```suggestion
      prompt = "Found existing config at " .. configFilePath .. ".\n Do you want overwrite it? (y/N):",
    }, function(input)
      if input == "y" then
```
One keypress is easier)